tox.ini

```diff
@@ -1,39 +1,37 @@
 [tox]
-envlist = py38-django{22,30,31,32}
+envlist = py38-django{32,40}
```

Comment:
We have added Django40 in tox but we are not using that in CI

Reply:
it was giving some error so will fix it in new PR.